tools/config/__init__.py

```diff
@@ -1204,7 +1204,8 @@ def validate_config(self):
                             min = int(str(min), 0) if min is not None else None
                             max = int(str(max), 0) if max is not None else None
 
-                            if (value < min or (value > max if max is not None else False)):
+                            if ((value < min if min is not None else False) or
```

Comment:
Could this be written:
```suggestion
                            if ((min is not None and value < min) or
```